src/pages/community/index.js

```diff
@@ -16,16 +16,22 @@ import {
   selectNthTraining,
 } from '../../components/training'
 import { LinkButton } from '../../components/buttons'
-import { MEETUP, selectMeetups, instagramPictures } from '../../config/data'
```

Comment:
we'll probably be able to totally get rid of the selectMeetups function as almost all of that data is now in Upmentoring